### DISCUSSION TITLE
what kind of image is best suited for paddleocr engine?

### DISCUSSION THREAD

**Discussion by vishal180618** [PaddlePaddle/PaddleOCR#14146] [Q&A] [answered]
Tesseract, suggests that images with black text on a white background should be best suited for OCR. I used the same in paddleOCR, but for some images, I have better results if the image has white text on a black background. 

So is there any official recommendation on what type of images should be fed to the paddle OCR engine?

Should I use black text on a white background or white text on a black background?

**Comment by GreatV** [comment, marked as answer]
PaddleOCR does not have any special restrictions and can handle combinations of various text and background colors. Although black text on a white background is usually the recommended format, it also adapts well to other high-contrast combinations, such as white text on a black background.

Usage Recommendations:
- Flexibility: Flexibly choose the text and background color combinations based on the characteristics of the actual image.
- Testing and Validation: Try different color combinations and observe the recognition effect in specific applications to find the best solution.
- Image Preprocessing: Appropriate image enhancement and preprocessing can further improve OCR effectiveness.

Overall, just choose the method that best suits your data and application scenario.
